desktop/core/src/desktop/js/reactComponents/FileChooserComponent/FileChooserComponentButton/FileChooserComponentButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './FileChooserComponentButton.scss';
+import FileChooserModal from '../FileChooserModal/FileChooserModal';
+import { Button } from 'antd';
+
+const FileChooserComponentButton = (): JSX.Element => {
+  const [show, setShow] = useState(false);
+
+  return (
+    <div>
+      <Button type="primary" onClick={() => setShow(true)}>
+        File chooser component
```

Comment:
I can imagine that we wanna reuse this button in multiple places, and possibly use different labels for the button, so you can add a prop this string. (In my upcoming i18n PR I will show how to combine i18n with props and default values)